pkg/kudoctl/cmd/version.go

```diff
@@ -30,5 +34,40 @@ func newVersionCmd() *cobra.Command {
 func VersionCmd(cmd *cobra.Command, args []string) error {
 	kudoVersion := version.Get()
 	fmt.Printf("KUDO Version: %s\n", fmt.Sprintf("%#v", kudoVersion))
+
+	// Print the Controller Version
+	controllerVersion, err := GetControllerVersion()
+	if err != nil {
+		fmt.Printf("KUDO Controller Version: %s\n", controllerVersion)
+	} else {
+		fmt.Printf("KUDO Controller Version: %#v\n", err)
+	}
+
 	return nil
 }
+
+// GetControllerVersion
+func GetControllerVersion() (string, error) {
+
+	controllerVersion := ""
+
+	client, err := kube.GetKubeClient(Settings.KubeConfig)
+	clog.V(3).Printf("Acquiring kudo client")
+	if err != nil {
+		clog.V(3).Printf("Failed to acquire kudo client")
+		return "", errors.New("<Failed to acquire kudo client>")
```

Comment:
Please return an error message that includes the underlying error. Also let's stay consistent with other error messages.
```suggestion
		return "", fmt.Errorf("failed to acquire Kubernetes client: %v", err)
```